Description: Avoid privacy leak in documentation
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2018-11-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/Data/YAML.hs
+++ b/src/Data/YAML.hs
@@ -116,9 +116,7 @@
 
 -- $overview
 --
--- The diagram below depicts the standard layers of a [YAML 1.2](http://yaml.org/spec/1.2/spec.html) processor. This module covers the upper /Native/ and /Representation/ layers, whereas the "Data.YAML.Event" and "Data.YAML.Token" modules provide access to the lower /Serialization/ and /Presentation/ layers respectively.
---
--- <<http://yaml.org/spec/1.2/overview2.png>>
+-- A [yaml.org spec diagram](http://yaml.org/spec/1.2/overview2.png) depicts the standard layers of a [YAML 1.2](http://yaml.org/spec/1.2/spec.html) processor. This module covers the upper /Native/ and /Representation/ layers, whereas the "Data.YAML.Event" and "Data.YAML.Token" modules provide access to the lower /Serialization/ and /Presentation/ layers respectively.
 --
 -- $start
 --
